docs/interpreter/jdbc.md

```diff
@@ -507,8 +507,9 @@ Here are some examples you can refer to. Including the below connectors, you can
     <td>hive_password</td>
   </tr>
   <tr>
-    <td>hive.proxy.user</td>
-    <td>true or false</td>
+    <td>default.proxy.user.property</td>
+    <td>hive.server2.proxy.user</td>
+  </tr>
 </table>
 
 Connection to Hive JDBC with a proxy user can be disabled with `hive.proxy.user` property (set to true by default)
```

Comment:
What I meant in above line was the key is `default.proxy.user.property` with an example value `hive.server2.proxy.user`; fair point, let me tweak this documentation. 

Yes, adding a log (logger.warn) does make sense, let me do it right away.